### PR TITLE
Retrieve all enabled versions

### PR DESCRIPTION
**Pull request by jhernand**
Currently the `cluster create` command retrieves only the first page of
enabled versions. This works fine in environments with less than 100
versions, but in the integration environment there are already 116
versions. This results in rejecting the command when the user tries to
use any of the newer versions that aren't in the first page. To avoid
that this patch changes the command so that it will always retrieve all
the versions, even if there are more than 100.

### CODE REVIEW

**Comment by jhernand** [comment]
@cblecker please review.

File: cmd/ocm/cluster/create/create.go [hunk 1]
```diff
@@ -164,3 +161,26 @@ func run(cmd *cobra.Command, argv []string) error {
...
+	size := 100
+	for {
+		var response *cmv1.VersionsListResponse
+		response, err = collection.List().
+			Search("enabled = 'true'").
+			Page(page).
+			Size(size).
+			Send()
+		if err != nil {
+			return
```
Comment (cblecker) [inline cmd/ocm/cluster/create/create.go]: stylistically, would it be better to not use naked returns?

Comment (jhernand) [reply cmd/ocm/cluster/create/create.go]: Well, I agree it is a matter of style/taste. I like them.

**Pull request merged**